src/components/organisms/ChatDrawer/ChatDrawer.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useContext, useEffect, useState } from "react";
+import MessageList from "components/molecules/MessageList";
+import CallOutMessageForm from "components/molecules/CallOutMessageForm";
+import { useForm } from "react-hook-form";
+import { ChatContext } from "components/context/ChatContext";
+import "./ChatDrawer.scss";
+import { useUser } from "hooks/useUser";
+import { useSelector } from "hooks/useSelector";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCommentDots,
+  faAngleDoubleLeft,
+} from "@fortawesome/free-solid-svg-icons";
+
+interface ChatOutDataType {
+  messageToTheBand: string;
+}
+
+const ChatDrawer = () => {
+  const roomName = "jazz";
+  const { user } = useUser();
+  const { usersById, chats } = useSelector((state) => ({
+    usersById: state.firestore.data.users,
+    chats: state.firestore.ordered.venueChats,
+  }));
+
+  const [isMessageToTheBarSent, setIsMessageToTheBarSent] = useState(false);
+  const [isChatDrawerExpanded, setIsChatDrawerExpanded] = useState(false);
+
+  useEffect(() => {
+    if (isMessageToTheBarSent) {
+      setTimeout(() => {
+        setIsMessageToTheBarSent(false);
+      }, 2000);
```

Comment:
why 2 second delay?

Reply:
instead of seeing "send" on the submit button, we see "Sent!" and the button is disabled for 2 seconds